src/marketing/engagement-cloud.md

```diff
@@ -15,8 +15,8 @@ Engagement Cloud, powered by [dotdigital][1]{: target="_blank"}, gives you the a
 * [Generate reports]({{ site.baseurl }}{%- link reports/customer-engagement.md -%})
 
 ## Changes In the Latest Release
```

Comment:
Add a blank line to separate the heading and text of the section.

Reply:
Fixed.